shared/featureconfig/config.go

```diff
@@ -223,6 +227,7 @@ func ConfigureValidator(ctx *cli.Context) {
 		log.Warn("Disabling new BLS library blst")
 		cfg.EnableBlst = false
 	}
+	cfg.KeystoreImportDebounceInterval = ctx.Int(dynamicKeyReloadDebounceInterval.Name)
```

Comment:
We can use ctx.Duration and have dynamicKeyReloadDebounceInterval to be a DurationFlag. 

https://pkg.go.dev/github.com/urfave/cli/v2#Context.Duration